src/msw/utils.cpp

```diff
@@ -1269,6 +1269,34 @@ wxWinVersion wxGetWinVersion()
     return wxWinVersion_Unknown;
 }
 
+wxString wxGetCpuArchitectureName()
+{
+    SYSTEM_INFO si;
+    GetNativeSystemInfo(&si);
+
+    switch (si.wProcessorArchitecture)
+    {
+    case PROCESSOR_ARCHITECTURE_AMD64:
+        return "x64";
+
+    case PROCESSOR_ARCHITECTURE_ARM:
+        return "ARM";
+
+    case 12: // PROCESSOR_ARCHITECTURE_ARM64
```

Comment:
It would be slightly preferable to use the actual symbol and then provide its definition, if it's not already defined, either near the top of this file (probably better in this case) or in `wx/msw/missing.h`.